### PR TITLE
Rename all Layer and Source properties 'projection' by 'crs'.

### PR DESCRIPTION
**Pull request by gchoqueux**
## Description
Rename all **`Layer`** and **`Source`** properties `projection` by **`crs`**.

**Deprecated**: **`Source.projection`** and **`Layer.projection`** properties are deprecated.
- Use **`crs`** instead of **`projection`**.
## Motivation
Have same **`crs`** projection property for : **`Extent, Coordinates, Features, Layer and Source`**

### CODE REVIEW

File: src/Layer/Layer.js [hunk 1]
```diff
@@ -75,6 +75,11 @@ class Layer extends THREE.EventDispatcher {
      * layerToListen.addEventListener('opacity-property-changed', (event) => console.log(event));
      */
     constructor(id, config = {}) {
+        /* istanbul ignore next */
+        if (config.projection) {
+            console.warn('Layer projection parameter is deprecated, use crs instead of.');
```
Comment (zarov) [inline src/Layer/Layer.js]: `use crs instead of` -> `use crs instead`.

File: src/Layer/OrientedImageLayer.js [hunk 2]
```diff
@@ -93,14 +93,19 @@ class OrientedImageLayer extends GeometryLayer {
...
      * it's a set of panoramic position and orientation.
      * @param { string } config.calibrations - Json object, representing a set of camera. see [CameraCalibrationParser]{@link module:CameraCalibrationParser}
      * @param { OrientedImageSource } config.source - Source used to build url of texture for each oriented image,
      * a tecture is need for each camera, for each panoramic.
      */
     constructor(id, config = {}) {
+        /* istanbul ignore next */
+        if (config.projection) {
+            console.warn('OrientedImageLayer projection parameter is deprecated, use crs instead of.');
+            config.crs = config.cr || config.projection;
```
Comment (zarov) [inline src/Layer/OrientedImageLayer.js]: Typo: `cr` -> `crs`

File: src/Source/FileSource.js [hunk 3]
```diff
@@ -100,19 +100,23 @@ const ext = new Extent('EPSG:4326', [0, 0, 0, 0]);
 class FileSource extends Source {
     /**
      * @param {Object} source - An object that can contain all properties of a
-     * FileSource and {@link Source}. Only `projection` is mandatory, but if it
-     * presents in `parsedData` under the property `projection` or `crs`, it is
-     * fine.
-     * @param {string} crsOut - The projection of the output data after parsing.
+     * FileSource and {@link Source}. Only `crs` is mandatory, but if it
+     * presents in `parsedData` under the property `crs`, it is fine.
+     * @param {string} crsOut - The crs projection of the output data after parsing.
```
Comment (zarov) [inline src/Source/FileSource.js]: Where is this parameter ?

File: src/Source/WFSSource.js [hunk 4]
```diff
@@ -92,18 +92,22 @@ import CRS from 'Core/Geographic/Crs';
...
-     * WFSSource and {@link Source}. `url`, `typeName` and `projection` are
+     * WFSSource and {@link Source}. `url`, `typeName` and `crs` are
      * mandatory.
      *
      * @constructor
      */
     constructor(source) {
+        if (source.projection) {
+            console.warn('WFSSource projection parameter is deprecated, use crs instead of.');
+            source.crs = source.cr || source.projection;
```
Comment (zarov) [inline src/Source/WFSSource.js]: `cr` -> `crs`

**Review by zarov — APPROVED** [review]
You missed some in this [tutorial](https://github.com/iTowns/itowns/blob/master/docs/tutorials/Display-a-geometry-layer.md)

**Comment by gchoqueux** [comment]
> You missed some in this tutorial

done, thx

File: src/Source/WFSSource.js (hunk 4 above)
Comment (gchoqueux) [reply src/Source/WFSSource.js]: done

File: src/Source/FileSource.js (hunk 3 above)
Comment (gchoqueux) [reply src/Source/FileSource.js]: removed

File: src/Layer/OrientedImageLayer.js (hunk 2 above)
Comment (gchoqueux) [reply src/Layer/OrientedImageLayer.js]: done

File: src/Layer/Layer.js (hunk 1 above)
Comment (gchoqueux) [reply src/Layer/Layer.js]: thx, done

**Pull request merged**